beast/observationmodel/ast/make_ast_xy_list.py

```diff
@@ -48,9 +48,8 @@ def pick_positions_from_map(
     Parameters
     ----------
 
-    catalog: GenFluxCatalog object
-        The output object from `datamodel.get_obscat(datamodel.obsfile,
-        datamodel.filters)`
+    catalog: Observations object
+        Provides the observations
 
     chosen_seds: astropy Table
```

Comment:
astropy Table -> astropy.table?

Reply:
Leaving it as is.  It really is astropy.table.Table, but that looks odd.  There is a way to reference the actual api for this class, but I haven't figured that out yet.  If I do, I'll use that method.  Googling now.

Reply:
Ok.  That looks hard.  Running away.